engine-jdk8-tests/src/test/java/org/hibernate/validator/test/internal/engine/valuehandling/JavaFXClassLoadingTest.java

```diff
@@ -20,6 +19,8 @@
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
 import org.junit.Test;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
```

Comment:
Ok, I'll update the IDE confing and make the changes to the files that are already affected by this commit. I'll push the changes once they are made. 
Also can you point me in the direction where to look why the build have failed ? I've checked the CI logs but cannot find anything useful there. Also I've tried to run locally and all seems to be working just fine, but it failed on a CI.